docs/widgets.md

```diff
@@ -475,3 +475,23 @@ cmake -DSLINT_STYLE="material" /path/to/source
 
 </details>
 
+### Selecting the Widget Style When Previewing Designs With `slint-viewer`
+
+Select the style either by setting the `SLINT_STYLE` environment variable, or passing the style name with the `--style` argument:
+
+```sh
+slint-viewer --style material /path/to/design.slint
+```
+
+### Selecting the Widget Style When Previewing Designs With The Slint Visual Studio Code Extension
+
+Select the style by first opening the Visual Studio Code settings editor:
+
+* On Windows/Linux - File > Preferences > Settings
+* On macOS - Code > Preferences > Settings
+
+Then enter the style name under Extensions > Slint > Preview:Style
+
+### Selecting the Widget Style When Previewing Designs With The Generic LSP Process
+
+Select the style by setting the `SLINT_STYLE` environment variable before launching the process.
```

Comment:
somehow the settings should work with some others LSP client.  It's just that there is no protocol in the LSP to define what settings are available, so someone needs to eddit the `slint.preview.style` setting somehow (i think we indeed should document our settings somewhere, like rust analizer does)

other than that, most editor also have a way to pass command line parameters to the language server which can also take a --style

Reply:
I've added a sentence for it - that's indeed an option.

Overall this just feels entirely wrong. I'm looking forward to having an actual UI in the preview, so that we can remove this part of the documentation again.

Reply:
Yes,  in this toolbar we are thinking of.